sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java

```diff
@@ -752,4 +752,58 @@ void testMetricStatusManipulationSETActivationStateNOTRDYBadSecondManipulationFa
         verify(mockManipulations).setComponentActivation(metricStateHandle, startActivationState);
         verify(mockManipulations).setMetricStatus(metricStateHandle, metricCategory, activationState);
     }
+
+    @Test
+    @DisplayName(
+            "testMetricStatusManipulationSETActivationStateFAIL: Set ActivationState " + "of all SET-Metrics to FAIL.")
```

Comment:
Why the concatenation?

Reply:
The string was supposed to be in 2 lines. Spotless reformatted it back into one line, which indeed looks a bit silly.
I will remove this.

Reply:
Done.